aws/db.tf

```
@@ -0,0 +1,63 @@
+resource "random_id" "db_name_suffix" {
+  byte_length = 4
+}
+
+resource "random_string" "postgres_airflow_password" {
+  count   = "${ var.postgres_airflow_password == "" ? 1 : 0 }"
+  length  = 8
```

```suggestion
  length  = 32
```

8 chars is a bit short :)